app/main.go

```diff
@@ -14,6 +14,8 @@ import (
 	"time"
 
 	"github.com/weaveworks/scope/xfer"
+
+	"github.com/PuerkitoBio/ghost/handlers"
```

Comment:
Please put third part imports above weaveworks/scope imports, and below standard library imports.